activesupport/lib/active_support/hash_with_indifferent_access.rb

```diff
@@ -289,6 +289,7 @@ def deep_stringify_keys; dup end
     undef :symbolize_keys!
     undef :deep_symbolize_keys!
     def symbolize_keys; to_hash.symbolize_keys! end
+    alias_method :to_options,  :symbolize_keys
```

Comment:
Nitpick: there only needs to be space between `:to_options,` and `:symnolize_keys`.

Reply:
👍